src/mesh.c

```
@@ -6,381 +6,315 @@
- */
-int mesh_process_packet(const packet_t *packet, int16_t rssi, int8_t snr)
-{
-    if (!packet)
-    {
-        return -1;
-    }
+/* ============================================================================
+ * Receive path: decrypt + process by type (beacon/handshake/data)
+ * ============================================================================ */
+
+int mesh_process_packet(const packet_t* packet, int16_t rssi, int8_t snr) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'mesh_process_packet' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)